bop_toolkit_lib/visualization.py

```diff
@@ -12,6 +12,8 @@
 from bop_toolkit_lib import inout
 from bop_toolkit_lib import misc
 
+from hand_tracking_toolkit.camera import CameraModel
```

Comment:
Explicit dependency on hand_tracking_toolkit was decided to be forbidden in this issue https://github.com/thodan/bop_toolkit/issues/134, which was fixed in https://github.com/thodan/bop_toolkit/pull/135. I order to be backward compatible (work with say ycbv) without needing to install hand_tracking_toolkit, the script should only issue a warning. You may follow a similar logic as here https://github.com/thodan/bop_toolkit/blob/e00ee3e8a38a77a201609a0f9a8136036b53d26c/scripts/eval_calc_errors.py#L28 or something similar.